elasticsearch/transferFiles.js

```diff
@@ -4,6 +4,8 @@ let logs, parsed;
 let logsToSend = [];
 const client = require('./index.js').client;
 const filePath = __dirname + '/logs.json';
+const path = require('path');
```

Comment:
where is 'path' being used?

Reply:
path will be used to allow this function to transfer files located in different directories, currently hacked together with commented out line--to be refactored